plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/util/PathComparator.groovy

```diff
@@ -0,0 +1,29 @@
+package com.here.gradle.plugins.jobdsl.util
+
+/**
+ * A comparator that orders path names. Files in the same directory are ordered alphabetically, and always come before
+ * files in subdirectories. Subdirectories are also ordered alphabetically.
+ */
+class PathComparator implements Comparator<String> {
+    @Override
+    int compare(String o1, String o2) {
+        def tokens1 = o1.tokenize('/')
```

Comment:
Shouldn't this use `File.separator` for portability?

Reply:
Done.

Reply:
I had to revert this as it fails on Windows.